charts/flyte-binary/values.yaml

```diff
@@ -81,6 +81,15 @@ configuration:
       gcs:
         # project Google Cloud project in which bucket resides
         project: "my-organization-gcp-project"
+      # azure Provider configuration for Azure object store
+      azure:
+        # configDomainSuffix Domain name suffix
+        configDomainSuffix: ""
+        # configUploadConcurrency Upload Concurrency (default 4)
+        configUploadConcurrency: 4
+        type: stow
```

Comment:
the `type` option isn't clear to me. I can't imagine it being able to be anything else.